### ISSUE TITLE
[Enhancement] Open currently playing track in a browser

### ISSUE THREAD

**Issue by eigenstil** [Soundnode/soundnode-app#417]
It would be good, if we could open the currently playing track in a browser.

In the stream view, there are buttons for liking, adding to playlist and opening the track in a browser:
![open track in browser](https://cloud.githubusercontent.com/assets/13272281/10689819/1b47168a-797e-11e5-8716-134c2b40a118.PNG)

this should also be added to the currently playing bottom bar, probably beneath the artist name (this would be the most consistent implementation to the stream view):
![implementation of open in browser](https://cloud.githubusercontent.com/assets/13272281/10689904/8bb04c8e-797e-11e5-97f3-ae61e20c1f28.PNG)

I think, this should be done together with #259, so it would look like this mockup:
![mockup-icon-bar](https://cloud.githubusercontent.com/assets/13272281/10690022/5b6c90cc-797f-11e5-8df6-f8d097ba47d5.jpg)

**Comment by mradionov** [comment]
Does adding a "Add to playlist" make sense though? If track is currently playing it is already in the playlist.

**Comment by eigenstil** [comment]
Yes, this makes perfect sense. The "Add to playlist" button doesn't add the song to the current playlist. It opens my personal playlists and then I can choose, to which playlist I want to add the selected song.

**Comment by eigenstil** [comment]
Why has this been closed? It isn't implemented. The only thing which was implemented is the heart-button.
